packages/smart-wallet/src/utils.js

```diff
@@ -19,16 +21,19 @@ export const makeWalletStateCoalescer = (invitationBrand = undefined) => {
    */
   const invitationsReceived = new Map();
 
-  /** @param {import('./smartWallet').UpdateRecord} updateRecord newer than previous */
+  /** @param {import('./smartWallet').UpdateRecord | {}} updateRecord newer than previous */
```

Comment:
I had better results doing `Partial<UpdateRecord>` rather than `UpdateRecord | {}` in `wallet-app` (where the source is `.ts`). Otherwise it would say `property "updated" doesn't exist on type {} blah blah blah` no matter what. If it works here that's cool though.